nac/offsets.go

```diff
@@ -168,4 +239,4 @@ func hexToByte32(val string) [32]byte {
 		panic(fmt.Errorf("expected 32 bytes, got %d", len(out)))
 	}
 	return *(*[32]byte)(out)
-}
+}
```

Comment:
Seems the newline at the end of the file got removed in this change.

Reply:
This should be fixed now.

Reply:
Thanks for fixing that up, looks good :)